register.php

```diff
@@ -28,6 +28,24 @@
 			}
 		}
 		
+		if(isset($_POST['g-recaptcha-response'])){
+          $captcha=$_POST['g-recaptcha-response'];
+        }
+		
+		if(!$captcha){
+          echo '<h2>Please check the the captcha form.</h2>';
+          exit;
+        }
+        $secretKey = "Put your secret key here";
```

Comment:
Indent the code properly, and the secret key variable should be placed in config.php.